cmake/usMacroCreateBundle.cmake

```diff
@@ -43,6 +43,22 @@ if(${PROJECT_NAME}_DEPENDS)
   )
 endif()
 
+# Set the resource embedding method
+set(_resource_embed_type )
+if(${PROJECT_NAME}_LINK_RESOURCES AND ${PROJECT_NAME}_APPEND_RESOURCES)
```

Comment:
Since there is only two possible states, should there be just one flag... something like "${PROJECT_NAME}_EMBED_TYPE" with values of either "LINK" or "APPEND". Then no error checking for and invalid state of two flags with 4 possible values (when only 2 are valid) needs to be done.

Reply:
@carneyweb This seems reasonable. If the value of the "embed resource type" is invalid it will be ignored by the macros which this one calls and the default resource embedding method will be used.